docs/content-editing/add-a-search-block.md

```diff
@@ -0,0 +1,431 @@
+    :target: /_images/add-a-search-block-step-23.jpeg
+    ```
+    +++
+    _Click on {guilabel}`Add Facet` to add another facet._
+    ````
+
+1.  Click the Label field to add a label to each facet.
+
+    Type "Date range" as you'll be adding a facet to filter by tag.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-24.jpeg
+    :alt: Click the Label field to add a label to each facet.
+    :target: /_images/add-a-search-block-step-24.jpeg
+    ```
+    +++
+    _Click the Label field to add a label to each facet._
+    ````
+
+1.  Click on {guilabel}`Field` to see the options available.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-25.jpeg
+    :alt: Click on `Field` to see the options available.
+    :target: /_images/add-a-search-block-step-25.jpeg
+    ```
+    +++
+    _Click on {guilabel}`Field` to see the options available._
+    ````
+
+1.  Click {guilabel}`Modification date`.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-26.jpeg
+    :alt: Click `Modification date`.
+    :target: /_images/add-a-search-block-step-26.jpeg
+    ```
+    +++
+    _Click {guilabel}`Modification date`._
+    ````
+
+1.  Click on {guilabel}`Facet widget` to pick which type of widget you want to use for this facet.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-27.jpeg
+    :alt: Click on `Facet widget` to pick which type of widget you want to use for this facet.
+    :target: /_images/add-a-search-block-step-27.jpeg
+    ```
+    +++
+    _Click on {guilabel}`Facet widget` to pick which type of widget you want to use for this facet._
+    ````
+
+1.  Select {guilabel}`Date Range` to display date picker widgets for a modification date range.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-28.jpeg
+    :alt: Select `Date Range` to display date picker widgets for a modification date range.
+    :target: /_images/add-a-search-block-step-28.jpeg
+    ```
+    +++
+    _Select {guilabel}`Date Range` to display date picker widgets for a modification date range._
+    ````
+
+1.  Click on "Sort on options".
+
+    This adds a widget with options for the user to sort their search.
+    
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-29.jpeg
+    :alt: Click on "Sort on options".
+    :target: /_images/add-a-search-block-step-29.jpeg
+    ```
+    +++
+    _Click on "Sort on options"._
+    ````
+
+1.  Click "Creation date" and "Effective date" to add them to the list of options to sort by.
+
+    ````{card}
+    ```{image} /content-editing/_static/add-a-search-block-step-30.jpeg
+    :alt: Click "Creation date" and "Effective date" to add them to the list of options to sort by.
+    :target: /_images/add-a-search-block-step-30.jpeg
+    ```
+    +++
+    _Click "Creation date" and "Effective date" to add them to the list of options to sort by._
```

Comment:
```suggestion
1.  Click {guilabel}`Creation date` and {guilabel}`Effective date` to add them to the list of options to sort by.

    ````{card}
    ```{image} /content-editing/_static/add-a-search-block-step-30.jpeg
    :alt: Click "Creation date" and "Effective date" to add them to the list of options to sort by.
    :target: /_images/add-a-search-block-step-30.jpeg
    ```
    +++
    _Click {guilabel}`Creation date` and {guilabel}`Effective date` to add them to the list of options to sort by._
```